syntax = 'proto2';
option optimize_for = LITE_RUNTIME;
import "Item.proto";
import "Computer.proto";

package DBToGS;

enum MsgID {
    eQueryAllAccount_CallBack = 1;
    eQueryUser_DBCallBack = 2;
}

message QueryAllAccount {
    optional MsgID id = 1 [default = eQueryAllAccount_CallBack];
    message Account {
        optional int32 id = 1;
        optional string nameGooglePlus = 2;
        optional int32 id_gps = 3;
        optional string nickname = 4;
    }
    repeated Account account = 2;
}

message Inventory {
    repeated ItemInfo item = 1;
}

message QueryUser {
    optional MsgID msgid = 1 [default = eQueryUser_DBCallBack];
    optional uint32 btc = 2;
    optional uint32 gcnetid = 3;
    optional bytes db = 4;
    repeated ComputerInfo computer = 5;
    required Inventory invenotry = 6;
}
